pages/index.js

```diff
@@ -1,14 +1,25 @@
 import React from 'react';
 
 import Hero from '../components/home/Hero';
-import Section from '../components/home/Section';
+import Welcome from '../components/home/Welcome';
+import Featured from '../components/home/Featured';
+import Conversion from '../components/home/Conversion';
+import Tools from '../components/home/Tools';
+import Quote from '../components/home/Quote';
+import Logos from '../components/home/Logos';
+import Bottom from '../components/home/Bottom';
 
 export default function Index() {
   return (
     <div>
-      <Section alt={1}>
```

Comment:
We could Add feature flag here to switch between old/new design

Reply:
instead of deleting the old component, use 
          
```
<Flags
   authorizedFlags={['feature-flag-name']}
   renderOn={() => (
            
   )}
   renderOff={() => (
  )}
```

Reply:
the document of the package we're using: https://www.npmjs.com/package/react-feature-flags